noxfile.py

```diff
@@ -103,6 +103,7 @@ def tests(session, shard):
         "--cov=weave",
         "--cov-report=html",
         "--cov-branch",
+        "-s",
```

Comment:
what does this do?

Reply:
it prevents pytest from hijacking stdin but it actually also results in all the test output getting printed to the screen rather than captured, which is not pretty so I will just pass this when I need to. closing this pr